app/backend/resources/datomic/ion-config.edn

```diff
@@ -55,9 +55,9 @@
            {:fn teet.gis.datasource-cleanup/cleanup-datasources-ion
             :description "Cleanup unreferenced deleted features"}
 
-           ;;:notify
-           ;; {:fn teet.notification.ion/notify
-           ;;  :description "Scheduled notifications"}
+           :notify
+            {:fn teet.notification.ion/notify
```

Comment:
the teet.notification.ion namespace should be renamed to teet.notification.notification-ion
and the file notification_ion.clj to comply with regular naming convention